CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Change `groupAllBy` to use a comparison function (#191)
```

Comment:
Probably don't even need this fixup note. Could just lump together with the entry for #182

Reply:
Add this PR to the changelog was preemptively tackled in #195